src/main/resources/config.properties

```diff
@@ -33,4 +29,14 @@ Axis.matlab.key = MATLAB
 Axis.use.matlab.warning = This project specifies MATLAB versions using the added 'MATLAB' axis as well as the 'Use MATLAB version' option. The value specified by 'Use MATLAB version' takes precedence over the values specified by the 'MATLAB' axis.
 Axis.no.installed.matlab.error = Because no MATLAB versions exist under Jenkins Global Tool Configuration, the plugin will run the default matrix configuration.
 Use.matlab.version.axis.warning = This project specifies MATLAB versions using the 'Use MATLAB version' option as well as the added 'MATLAB' axis. The value specified by 'Use MATLAB version' takes precedence over the values specified by the 'MATLAB' axis.
+matlab.tools.auto.install.display.name = Install Using MPM
```

Comment:
I think it's important to spell out MPM here:

Install Using MATLAB Package Manager

Reply:
Done